README.md

```diff
@@ -2086,6 +2087,31 @@ params do
 end
 ```
 
+### Using `dry-validation` or `dry-schema`
+
+As an alternative to the `params` DSL described above, you can use a schema or `dry-validation` contract to describe an endpoint's parameters. This can be especially useful if you use the above already in some other parts of your application. If not, you'll need to add `dry-validation` or `dry-schema` to your `Gemfile`.
+
+Then call `contract` with a contract or schema defined previously
+
+```rb
+contract CreateFoosSchema
```

Comment:
Maybe include the code for `CreateFoosSchema` or at least the class declaration.

A different example that "Foos" would be nice, something that everybody agrees is a thing.

Reply:
Added, see the new push.